dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -333,52 +281,53 @@ fi
 #
 if [[ "${GITPOD_WITH_DEDICATED_EMU}" != "true" ]]
 then
-  kubectl --kubeconfig "${DEV_KUBE_PATH}" --context "${DEV_KUBE_CONTEXT}" -n werft get secret stripe-api-keys -o yaml > stripe-api-keys.secret.yaml
-  yq w -i stripe-api-keys.secret.yaml metadata.namespace "default"
-  yq d -i stripe-api-keys.secret.yaml metadata.creationTimestamp
-  yq d -i stripe-api-keys.secret.yaml metadata.uid
-  yq d -i stripe-api-keys.secret.yaml metadata.resourceVersion
-  diff-apply "${PREVIEW_K3S_KUBE_CONTEXT}" stripe-api-keys.secret.yaml
-  rm -f stripe-api-keys.secret.yaml
-
-  yq w -i "${INSTALLER_CONFIG_PATH}" "experimental.webapp.server.stripeSecret" "stripe-api-keys"
```

Comment:
@iQQBot I had to add this back, so that we can use UBP in preview environments. Ref: #19692 .